src/app/not-found.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from "next/link";
+
+import { BackButton } from "@/components/BackButton";
+import { UabcHeaderText } from "@/components/UabcHeaderText";
+import { UabcLogoNotFound } from "@/components/UabcLogoNotFound";
+import { Button } from "@/components/ui/button";
+
+const page = () => {
+  return (
+    <div className="h-dvh w-dvw flex flex-col justify-evenly items-center overflow-hidden bg-background">
+      <UabcHeaderText />
+      <div>
+        <div className="w-72 aspect-square grid place-items-center">
+          <UabcLogoNotFound className="absolute min-w-[250px]" />
+          <h1 className="text-9xl font-bold text-foreground z-10 select-none">
+            404
```

Comment:
optional: not sure how you could do this but a while outline around the 404 could be nice